sdk/helper/certutil/helpers.go

```diff
@@ -620,15 +657,25 @@ func CreateCertificate(data *CreationBundle) (*ParsedCertBundle, error) {
 
 var oidExtensionBasicConstraints = []int{2, 5, 29, 19}
 
-// Creates a CSR. This is currently only meant for use when
-// generating an intermediate certificate.
+// CreateCSR creates a CSR with the default rand.Reader to
+// generate a cert/keypair.
```

Comment:
Don't lose the bit about "This is currently only..."

Reply:
Done!